.github/workflows/e2e.yml

```diff
@@ -0,0 +1,25 @@
+name: e2e
+on: [pull_request]
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      # Install Node.js
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: yarn install
+      
+      - name: Prepare server
+        run: docker-compose -f docker-compose.e2e.yml up -d
```

Comment:
Would [something like this](https://semaphoreci.com/docs/docker/docker-layer-caching.html) work to pull the :latest image from master and then use that as a cache to build the image?